atomic_reactor/plugins/pre_download_remote_source.py

```diff
@@ -84,58 +125,61 @@ def run(self):
 
         session = get_retrying_requests_session()
 
-        # Download the source code archive
+        self.new_remote_sources = self.remote_sources[0]['name'] is not None
```

Comment:
one alternative idea might be to check _self.workflow.source.config.remote_sources_ instead of using self.new_remote_sources.

We could possibly also issue warning/raise error based on _self.workflow.source.config.remote_sources_ if user is using remote_sources but used  COPY $REMOTE_SOURCE $REMOTE_SOURCE_DIR in the dockerfile.

Reply:
sure I'll just use self.workflow.source.config.remote_sources

Reply:
we don't have to check dockerfile for that, since those variables won't be defined anyway so it won't work anyway